packages/storybook-vue/stories/components/card/Card.stories.mdx

```diff
@@ -55,16 +55,20 @@ export const Template = (args, { argTypes }) => ({
             >
                 <div v-if="slot.startsWith('<div')">
                     <div style="margin: -24px;">
-                      <img style="margin-bottom: -4px; width:100%;" src="assets/card-grayscale.jpeg" alt="placeholder" />
+                      <img style="width:100%;" src="assets/card-image.jpg" alt="placeholder" />
```

Comment:
We also need the `display: block` on the image (otherwise it has that ~4px space in the bottom)  🥸 🙏 — also the markup in the snippet needs to be updated…